src/Pulsar.Client/Internal/TableViewImpl.fs

```diff
@@ -8,12 +8,14 @@ open Pulsar.Client.Api
 
 open System
 
-type internal TableViewImpl<'T> private (reader: IReader<'T>) =
+type internal TableViewImpl<'T when 'T: null> private (reader: IReader<'T>) =
```

Comment:
That's going to be breaking change (if some one has been passing structs like int or float as message value)

Reply:
Reverted.